components/fsxa/sections/InteractiveYoutubeVideo.tsx

```diff
@@ -0,0 +1,56 @@
+import { Component } from 'vue-property-decorator'
+
+import { RichTextElement } from 'fsxa-api/dist/types'
```

Comment:
I'm not quite sure if the import from `dist` is the right place from which we should import RichTextElement

Reply:
Stange... some components have it like this (product.tsx, homepage.tsx), others use just `fsxa-api`. Is there really a difference?